DDECal/test/integration/foursources.reg

```diff
@@ -0,0 +1,21 @@
+# Region file format: DS9 version 4.1
+global color=green dashlist=8 3 width=1 font="helvetica 10 normal roman" select=1 highlite=1 dash=0 fixed=0 edit=1 move=1 delete=1 include=1 source=1
+fk5
+
+# The polygons below were created manually. They create areas around each of the image points.
+# The comments below contain the corresponding 0-based pixel coordinates.
+
+# Ra+dec@(800,128). Polygon is (726, -1) (720, 298) (1061, 313) (1078, 14).
+polygon(22.0,28.0, 22.0,31.0, 18.0,31.0, 18.0,28.0)
```

Comment:
Depending on which formats we want to support, maybe good to stick to the example format that André gave, i.e. with the text label. We'll need that later on.